### PR TITLE
UI work again

### PR DESCRIPTION
**Pull request by instMeyFTW**
Please approve and merge

### CODE REVIEW

**Review by EngineeringDude123 — APPROVED** [review]
Approved

**Review by NoahBartell — APPROVED** [review]
Lgtm

File: dunwoody_disaster/CharacterFactory.py [hunk 1]
```diff
@@ -267,10 +267,10 @@ def Cooper() -> Character:
 
         char.weapons = [
             Item.Weapon(
-                name="Syntax Error", damage=10, magic=5, stamina=5, magical=True
+                name="Syntax\nError", damage=10, magic=5, stamina=5, magical=True
```
Comment (chanson02) [inline dunwoody_disaster/CharacterFactory.py:270]: `label.setWordWrap(True)` 🤯

File: dunwoody_disaster/views/MapScreen.py [hunk 2]
```diff
@@ -160,7 +160,7 @@ def buildMap(char: Character) -> "Map":
         map.addRoom("Black 42", (1092, 130), chars.RyanRengo(), "Courtyard+")
         map.addRoom("Math", (540, 570), chars.NoureenSajid(), "MathClass+")
         map.addRoom(
-            "Center of Engineering Excellence",
+            "Center of Engineering\nExcellence\n",
```
Comment (chanson02) [inline dunwoody_disaster/views/MapScreen.py:163]: Why end with a new line???

**Review by chanson02 — COMMENTED** [review]
I have several questions.

**Pull request merged**